content/TrueCommand/Reports/Creation.md

```diff
@@ -14,43 +14,51 @@ The **Reports** page customizes system metrics charts for data analysis.
-  `Stack the values` can be set to bring data points on the chart closer together.
-  This setting is useful for charts that have many different data points at the max Y value.
-* Data sources: Add data sources to the chart by expanding a category and selecting appropriate sources.
-  Multiple data sources can be added to one chart.
-* Summary: This step shows the all of the chosen values.
-  Click *SAVE* to add the custom chart to the report or *BACK* to go back and change a setting or data source.
+  You can set `Stack the values` to bring data points on the chart closer together.
+  `Stack the values` is helpful for charts with many different data points at the max Y value.
+* Data sources: Add data sources to the chart by expanding a category and selecting which sources you want.
+  You can add multiple data sources to one chart.
+* Summary: This step shows all chosen values.
+  Click **SAVE** to add the custom chart to the report, or **BACK** to change a setting or data source.
 
 After adding charts to the report, click **SAVE** to make this report available for use.
 
 
 ![ReportsCreateReportSave](/images/TrueCommand/2.0/ReportsCreateReportSave.png "Reports Create Report Save")
 
-After creating a report, you can click **GENERATE** to [generate the report]({{< relref "Generation.md" >}}) or you can go back to the reports page and make create another report.
+After creating a report, you can click **GENERATE** to [generate the report]({{< relref "Generation.md" >}}), or you can go back to the reports page and create another.
 
 ![ReportsGenerate](/images/TrueCommand/2.0/ReportsGenerate.png "Reports Generate")
 
+## Edit a Report
+
+Open the Reports page and click <i class="material-icons" aria-hidden="true" title="edit">edit</i> next to the report name to rename a report, add a description, or add/delete a Widget.
+
 ## Share Report
 
-By default, the reports created by a user are available only to that user.
-To share a report with other users or teams, open the Reports page and click the <mat-icon _ngcontent-fxr-c199="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">share</mat-icon> icon for the chart.
+By default, user-created reports are available only to that user.
+To share a report with other users or teams, open the **Reports** page and click the <mat-icon _ngcontent-fxr-c199="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">share</mat-icon> icon for the chart.
 
 ![ReportsSharing](/images/TrueCommand/2.0/ReportsShareReport.png "Reports Sharing")
 
 You can share reports with individual users or entire teams.
+
+## Delete a Report
+
+Delete a report by clicking <i class="material-icons" aria-hidden="true" title="Delete">delete</i>. This will permenently delete the report but it it can be recreated as needed.
```

Comment:
Remove will from sentence, change to this permanently deletes and correct the spelling of permanently. Suggest changing 'but it can be' to but you can recreate it as needed.